### PR TITLE
Add videogular

### PR DESCRIPTION
**Pull request by CodingCarlos**
- Included videogular to project
 - Created player directive and included in download and video detail.
 - Updated video card image error detection.

### CODE REVIEW

File: components/video-card/video-card.scss [hunk 1]
```diff
@@ -44,6 +44,11 @@ video-card {
 			}
 		}
 	}
+
+	.pixel {
+		height: 0px;
+		width: 0px;
+	}
```
Comment (alvaromart) [inline components/video-card/video-card.scss:51]: Why this .pixel height: 0, width:0 is necessary? I would like to learn.

**Review by alvaromart — COMMENTED** [review]
If we have added videogular to platform, maybe it is needed to add their use and license in some place. If we don't have any place yet we could start it as good practice.

File: components/video-card/video-card.scss (hunk 1 above)
Comment (CodingCarlos) [reply components/video-card/video-card.scss:51]: Yes, that's key. 

In old versions, we used to have an `` for the poster. But that was a big problem when the image size was different than expected. So, we setted an empty box, and used the poster as a background image, which will never be bigger than the actual image. But that don't allow to check image errors, as the image tag did.
  
What I've done is adding a 0px x 0px image with the poster, just to be check if there are an error. If it does, the `onerror` attribute of that image will be fired, and the background image of the parent box will be setted to the error image.

If we don't do this, we would have images with random sizes again, or deformed images, and the ugly "error image" that browser uses if there is an error.

**Comment by CodingCarlos** [comment]
As Videogular have MIT license, is not mandatory to do that. But yes, would be a great idea to start listing every resource we use and its license.

**Review by javierLiarte — APPROVED** [review]
seems all ok. Also tested in my computer and apart from a problem with thumbnail sizing, seems to work ok.

Approving to merge after solving thumbnail sizing error

**Comment by javierLiarte** [comment]
The video thumbnail doesnt fill player view 

image

**Comment by CodingCarlos** [comment]
Actually, there's not a thumbnail size error, but a global width size error. It was calculating size with scrollbars. Anyway, it's fixed ;) @javierLiarte

**Pull request merged**